### ISSUE TITLE
Decorators

### ISSUE THREAD

**Issue by matthewp** [stealjs/steal#1039]
Decorators are a Stage 2 feature that will soon be in Stage 3. Many libraries are coming to rely on them. I think we should include them with our Babel build.

**Comment by justinbmeyer** [comment]
Maybe as a related issue, do we know what it takes to customize babel?  Is that possible/documented/easy?

**Comment by matthewp** [comment]
Do you mean to add plugins as a user of Steal? Currently that's not possible. Probably should have a separate issue for that.

**Comment by justinbmeyer** [comment]
@matthewp yeah, that's what I mean. https://github.com/stealjs/steal/issues/1040

**Comment by matthewp** [comment]
## Goals

* We would like to be able to easily use decorators in Steal projects. 

## Implementation

Depending on how https://github.com/stealjs/steal/issues/1040 goes, there might not need to be anything here at all. If the 3rd party decorator plugin works after #1040, this issue can be closed.

However, if it doesn't work because of any of these reasons:

* It tries to load `fs` or something that can't be done in a browser.
* It causes a bunch of 404s because it uses the `folder` for `folder/index` convention.
* It's super slow because it loads a bazillion dependencies.

Then I think we should browserify/webpackify/stealify the decorator plugin and include it ourselves. Maybe somewhere like `steal/ext/babel/decorators.js`.

**Comment by matthewp** [comment]
@m-mujica Since decorators are included with babel-standalone, to complete this issue we just need a test.

**Comment by m-mujica** [comment]
Example app: https://github.com/m-mujica/steal-decorators-example

need to fix the build, it's broken for plugins included in the babel-standalone build but not supported by `transpile`.

**Comment by m-mujica** [comment]
Example app updated https://github.com/m-mujica/steal-decorators-example; it builds fine now.
